lib/controller.js

```diff
@@ -677,6 +677,32 @@ controller.connectToNetwork = function(opts) {
       if (security && securityOptions.indexOf(security) < 0) {
         return reject(`"${security}" is not a valid security option. Please choose on of the following: ${securityOptions.join(', ')}`);
       }
+      
+      if (security === 'wep') {
+        // WEP passphrases can be 10, 26, or 58 hexadecimal digits long.
+        // Match for hexadecimal characters:
+        const regex = /^[0-9a-f]+$/i;
+        if (!regex.test(password)) {
```

Comment:
Instead of regex parsing, you could alternatively check if the value is a valid hexadecimal number:

```js
isNaN(`0x${password}`)
```